assets/src/components/TableAssignment.js

```diff
@@ -125,8 +125,8 @@ function CustomAssignmentTable (props) {
       return row
     })
 
-  const tableRow = (row, i) => (
-    <TableRow key={i}>
+  const tableRow = (row, i, isCurrentWeek) => (
+    <TableRow key={i} {...(isCurrentWeek ? { ref: currentWeekRow } : {})}>
```

Comment:
Same deal in this table component. I'm replacing `RootRef`  by just applying the `tableRef` to `Table` and then using this logic so that the scrolling to the current week still works.

Reply:
This working nicely. thanks for noticing that this was deprecated.